build.gradle

```diff
@@ -15,7 +15,7 @@ plugins {
 ext {
     minSdkVersion = 24
     compileSdkVersion = 34
-    targetSdkVersion = 34
+    targetSdkVersion = 33
```

Comment:
Comes from https://github.com/wordpress-mobile/WordPress-Android/pull/20964, so it's expected to be part of the backmerge PR 👍